packages/frontend-common/formats/chart/hierarchical-tree/useFormatTreeData.spec.tsx

```diff
@@ -0,0 +1,629 @@
+import { renderHook } from '@testing-library/react';
+import type { Datum } from './type';
+import { useFormatTreeData } from './useFormatTreeData';
+
+describe('useFormatTreeData', () => {
```

Comment:
For all test on useFormatTreeData please test the whole tree result, instead of part of it.